reconcile/ocm_upgrade_scheduler.py

```diff
@@ -292,18 +350,24 @@ def act(dry_run, diffs, ocm_map):
     for diff in diffs:
         action = diff.pop("action")
         cluster = diff.pop("cluster")
+        gates_to_agree = diff.pop("gates_to_agreee")
         ocm = ocm_map.get(cluster)
         if action == "create":
             logging.info([action, cluster, diff["version"], diff["next_run"]])
             if not dry_run:
+                if len(gates_to_agree) > 0:
+                    agreement = ocm.create_version_agreement(gates_to_agree)
```

Comment:
It would be worth catching any exception here (even if only to re-raise it) and prepend `f"[{cluster}] "` in the message for easier analysis in case of problem creating the versiongate agreement